CRM/Core/BAO/CustomValueTable.php

```diff
@@ -360,7 +360,10 @@ public static function store($params, $entityTable, $entityID, $parentOperation
           'custom_group_id' => $customValue['custom_group_id'],
           'table_name' => $customValue['table_name'],
           'column_name' => $customValue['column_name'],
+          // is_multiple and serialize are related but different:
+          // is_multiple refers to the custom group, serialize to the field.
```

Comment:
Much like every college essay I ever wrote, if you delete the first half which is just filler, you're left with the good part that makes an actual point. :)
```suggestion
          // is_multiple refers to the custom group, serialize to the field.
```

Reply:
He he ... fair enough.